script/release.js

```diff
@@ -451,4 +453,25 @@ async function validateChecksums (validationArgs) {
     `shasums defined in ${validationArgs.shaSumFile}.`)
 }
 
+async function tagLibCC () {
+  const tag = `electron-${pkg.version}`
+  const libccDir = path.join(path.resolve(__dirname, '..'), 'vendor', 'libchromiumcontent')
+  console.log(`Tagging release ${tag}.`)
+  let tagDetails = await GitProcess.exec([ 'tag', '-a', '-m', tag, tag ], libccDir)
+  if (tagDetails.exitCode === 0) {
+    let pushDetails = await GitProcess.exec(['push', '--tags'], libccDir)
+    if (pushDetails.exitCode === 0) {
+      console.log(`${pass} Successfully tagged libchromiumcontent with ${tag}.`)
+    } else {
+      console.log(`${fail} Error pushing libchromiumcontent tag ${tag}: ` +
+        `${pushDetails.stderr}`)
+      process.exit(1)
```

Comment:
Should failed tagging fail the whole release process?
I mean it's a bad thing for sure, but not critical.

Reply:
@alexeykuzmin I debated this one and decided I want it to fail because it is then obvious it didn't work.  If it was just a log message we might miss it on a release.  You are right though that it isn't critical.

Reply:
And it can be corrected post release

Reply:
@alexeykuzmin I decided to update the script to not fail the whole release process.